tensorflow/contrib/android/java/org/tensorflow/contrib/android/TensorFlowInferenceInterface.java

```diff
@@ -194,6 +194,13 @@ public void run(String[] outputNames) {
    * @param outputNames A list of output nodes which should be filled by the inference pass.
    */
   public void run(String[] outputNames, boolean enableStats) {
+    run(outputNames, false, new String[] {});
```

Comment:
did you mean to say run(outputNames, enableStats, new String[] {})?

Reply:
Yes. Fixed in the following commit